ads/adverticum.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript} from '../3p/3p';
+import {validateData} from '../3p/3p';
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function adverticum(global, data) {
+  validateData(data, ['goa3zone'], ['costumetargetstring']);
+  const zoneid = 'zone' + data['goa3zone'];
+  const d = global.document.createElement('div');
+
+  d.setAttribute('id', zoneid);
+  d.setAttribute('class', 'goAdverticum goa-embedded');
+
+  document.getElementById('c').appendChild(d);
+
+  const s = global.document.createTextNode(data['costumetargetstring']);
```

Comment:
just fyi: `data['constumetargetstring']` is optional. I see you checked `if(data.costumetargetstring)` before, but removed that check since last commit. So it is possible to create an "undefined" textNode now, is this expected?